packages/test-kit/src/with-feature.ts

```diff
@@ -272,7 +280,13 @@ export function withFeature(withFeatureOptions: IWithFeatureOptions = {}) {
                 page.setDefaultNavigationTimeout(30_000);
                 page.setDefaultTimeout(10_000);
                 page.on('pageerror', (e) => {
-                    capturedErrors.push(e);
+                    if (
+                        !allowedErrors.some((allowed) =>
+                            allowed instanceof RegExp ? allowed.test(e.message) : e.message.includes(allowed)
```

Comment:
Readability is more important than performance here

Reply:
`e.message.includes(allowed)`, but "strings are tested for exact match" in the API comment?

Reply:
My bad will verify equality. Nice catch.